index.rst

```diff
@@ -366,3 +371,4 @@ Parallel and Distributed Training
    intermediate/dist_tuto
    intermediate/rpc_tutorial
    beginner/aws_distributed_training_tutorial
+   intermediate/rpc_param_server_tutorial
```

Comment:
does the order here matter?

Reply:
@mrshenli it makes this bottom one show in the bottom right of the page, like this:
<img width="871" alt="pt_tut" src="https://user-images.githubusercontent.com/8039770/79027382-df23a500-7b40-11ea-8105-57c5c28134ae.png">